config.ini

```diff
@@ -25,9 +25,9 @@ period                  = day
 
 [Tr2Vec]
 window_size     = 10
-mcc_vocab_size  = 344
+mcc_vocab_size  = 345
 mcc_emb_size    = 16
-amnt_bins       = 50
+amnt_bins       = 51
```

Comment:
Why we need 51 bins when we are masing only mccs?

Reply:
I masked amount too

Reply:
Yeah, I am stupid)))